doc/how-to-build.md

```diff
@@ -109,6 +109,49 @@ Makefile rules:
 Environment variable can be used to configure APISIX. Please take a look at `conf/config.yaml` to
 see how to do it.
 
+### Troubleshooting
+**Compile and install luasocket**
+
+If you run `make init` or `make run` to get an issue `Symbol not found: _luaL_openlib`, make sure to use the right version of luaSocket and lua. You can change lua 5.3.5 using LuaJIT or lua 5.1, if it still doesn't work, you can manually compile and install luasocket.
```

Comment:
The English version is different from the Chinese one?

Reply:
I think the `makefile` has processed this problem. so we call `make deps` should be enough for users.

https://github.com/apache/apisix/blob/master/Makefile#L55

Reply:
@spacewander It should be unclear for my expression, I will modify it

Reply:
@membphis This problem occurs after `make deps`,  I will complete the detail and environments

Reply:
what is the version of your Lua and Luarocks? @fukiki

Reply:
@membphis macosx  Lua 5.3.5  Luarocks 3.4.0